repos/spack_repo/builtin/packages/aretomo2/package.py

```diff
@@ -0,0 +1,56 @@
+# Copyright Spack Project Developers. See COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+from spack_repo.builtin.build_systems.cuda import CudaPackage
+from spack_repo.builtin.build_systems.makefile import MakefilePackage
+
+from spack.package import *
+
+
+class Aretomo2(MakefilePackage, CudaPackage):
+    """AreTomo2 is a multi-GPU accelerated software package that fully automates
+    motion-corrected marker-free tomographic alignment and reconstruction, now
+    includes robust GPU-accelerated CTF (Contrast Transfer Function) estimation
+    in a single package. AreTomo2 is part of our endeavor to build a
+    fully-automated high-throughput processing pipeline that enables real-time
+    reconstruction of tomograms in parallel with tomographic data collection.
+    It strives to be fast and accurate, as well as provides for easy
+    integration into subtomogram processing workflows by generating IMod
+    compatible files containing alignment and CTF parameters needed to
+    bootstrap subtomogram averaging. AreTomo2 can also be used for on-the-fly
+    reconstruction of tomograms and CTF estimation in parallel with tilt series
+    collection, enabling real-time assessment of sample quality and adjustment
+    of collection parameters."""
+
+    homepage = "https://github.com/czimaginginstitute/AreTomo2"
+    url = "https://github.com/czimaginginstitute/AreTomo2/archive/refs/tags/v1.1.2.tar.gz"
+
+    maintainers("Markus92")
+
+    license("BSD-3-Clause", checked_by="Markus92")
+
+    version("1.1.2", sha256="4cbb4d25d28778041d80ef2c598519b17b9a40aa84e1e99daf48ad5a90d946b4")
+
+    depends_on("c", type="build")
+    depends_on("cxx", type="build")
+    depends_on("gmake", type="build")
+
+    depends_on("cuda")
```

Comment:
This is not needed if the package is a `CudaPackage`:
```suggestion
```
you might also want to redefine the `cuda` variant so that its default is `True`.